js/utils.js

```diff
@@ -771,6 +771,8 @@ const CombodoGlobalToolbox = {
  * @since 3.0.0
  */
 const CombodoTooltip = {
+	
+	aTooltipRemoteContent: {},
```

Comment:
```suggestion
	/**
	 * @var aTooltipRemoteContent {Object}
	 * @since 3.1.0
	 */
	aTooltipRemoteContent: {},
	
```

Detail what its purpose is.
Also plural.

Reply:
Done in [20d5e65](https://github.com/Combodo/iTop/pull/476/commits/20d5e657346868a93b265edaccb5c8acde03e664)